include/mgp.hpp

```diff
@@ -30,9 +30,19 @@
 #include "_mgp.hpp"
 #include "mg_exceptions.hpp"
 #include "mg_procedure.h"
+#include "storage/v2/id_types.hpp"
```

Comment:
@DavIvek this won't work under MAGE because it's memgraph's internal storage hpp file. MAGE C++ API shouldn't be dependent in any way on the Memgraph's private storage implementation.

When building MAGE it breaks in the following way:
<img width="1652" alt="Screenshot 2024-10-25 at 7 13 02 PM" src="https://github.com/user-attachments/assets/7e58a6b5-82de-493d-8f1b-3f48add4b834">

Reply:
"Solved" under https://github.com/memgraph/memgraph/pull/2406/commits/f90221aa63c046a6457d64bc556ca59f7b77946f 😃